config/twill.php

```diff
@@ -332,6 +338,15 @@
      */
     'bind_exception_handler' => false,
 
+    'capsules' => [
+        'list' => [
+            [
+                'name' => 'EducatorResource',
+                'enabled' => true,
+            ],
+        ],
+    ],
+
```

Comment:
Capsules are third-party Twill extensions, so this config would imply that we installed a Twill extension called `EducatorResource`. I don't see anything like that in this PR, so I think this `'capsules'` config can be removed.